package.json

```diff
@@ -64,7 +64,7 @@
   ],
   "devDependencies": {
     "@types/chai": "^4.2.14",
-    "@types/react-native": "^0.70.6",
+    "@types/react-native": "^0.70.13",
```

Comment:
I don't see a 0.70.13 for @types/react-native. See [here](https://www.npmjs.com/package/@types/react-native?activeTab=versions).